### PR TITLE
Feat/saving tab layout to a file

### PR DESCRIPTION
**Pull request by alekspickle**
just to see the diff

### CODE REVIEW

File: zellij-server/src/screen.rs [hunk 1]
```diff
@@ -1807,6 +1809,17 @@ pub(crate) fn screen_thread_main(
...
+            ScreenInstruction::DumpLayout(client_id, layout) => {
+                let layout: Vec<(&Tab, Vec<PaneId>)> = screen
+                    .tabs
+                    .values()
+                    .map(|t| (t, t.get_all_pane_ids()))
+                    .collect();
+                let tab_names: Vec<String> = layout.iter().map(|(t, _)| t.name.clone()).collect();
+                println!("TABS: {:?}", tab_names);
+                screen.unblock_input()?;
+                screen.render()?;
```
Comment (alekspickle) [inline zellij-server/src/screen.rs:1821]: nevermind this

**Comment by Lockszmith-GH** [comment]
Sorry to burst in. If you do not find this helpful, I apologize for assuming this wasn't known.

You stated 'just to see the diff' - FYI if you use vscode, you can use [Git Tree Compare](https://marketplace.visualstudio.com/items?itemName=letmaik.git-tree-compare) extensions to compare branches.

If you are purely command line centered, after setting up [vimdiff as difftool](https://vi.stackexchange.com/a/626/42595), I use the following __git alias__:
```
git config  --global 'alias.dt' 'f() { git diff --stat "${@}" && git difftool "${@}"; }; f'
```

And then to compare my branch with main:
```
git dt origin/main
```

or to compare with a specific commit, use it's SHA hash (example using current main)
```
git dt df0c7f1
```

**Comment by alekspickle** [comment]
@Lockszmith-GH thx for info, but it's not meant to see the diff for me :)

**Comment by Lockszmith-GH** [comment]
In Ubuntu vimdiff is shipped with Neo-Vim, if for some reason it is not, see this [vi-stackexchange answer](https://vi.stackexchange.com/a/626/42595) for all the details you might need.

**Comment by imsnif** [comment]
@alekspickle - we're looking at this PR, right? To me it says "Closed" and not merged... I also double-checked with main and I don't see it merged. Maybe indeed github being a little funny? what do you see?

**Comment by imsnif** [comment]
I have changes locally (adding plugins to the mix) which I can't push right now... Github tells me you did a force push?
Normally I'd say let's scrap everything and make a new PR, but I'm a little afraid the two of you will lose your very well deserved credit on this.

**Comment by alekspickle** [comment]
as I said, i did not force push, and  I could rebase with tagging Alex manually, but it seems they commited [without metadata whatsoever](https://github.com/AlixBernard/zellij/commit/502fdc07ca85ab12d33c6ca595f85ab83aba8ce7) so if you could provide me with your github email I could do it, otherwise I see the only way is to create new PR

**Comment by AlixBernard** [comment]
I guess I should have signed my commit?
My GitHub email is alix.bernard9@gmail.com, I hope you can resolve the issue

**Comment by imsnif** [comment]
Alright, I'm honestly not sure what happened here - but I guess we can jot this down to one of those git mysteries. I opened a new branch/PR that we can work on here and it seems okay to me: https://github.com/zellij-org/zellij/pull/2801

**Comment by alekspickle** [comment]
I'm not crazy, GitHub is :stuck_out_tongue: 
Screenshot_20230920_213548

**Comment by imsnif** [comment]
Just to be clear, I never thought you were crazy and will believe you over Github any day of the week :)

**Pull request closed without merging**